Assets/Scripts/Character/Special Moves/Scarf.cs

```diff
@@ -13,6 +11,7 @@ private set
         {
             active = value;
             Animator.SetBool(ScarfHash, active);
+            wormth.SlowChill(active);
```

Comment:
This should access `wormth.multiplier` directly and change it based on the scarf type